Tools/Plotfile/CMakeLists.txt

```diff
@@ -27,6 +27,11 @@ foreach( _exe IN LISTS _exe_names)
    if (AMReX_CUDA)
       set_source_files_properties(${_exe}.cpp PROPERTIES LANGUAGE CUDA)
    endif()
+
+    # Add suffix to each tool's name to make them unique when installed.
+    # This avoids potential collisions of names on user systems, e.g., in
+    # software packages (Spack/Conda/Debian/...).
+    set_target_properties(${_exe} PROPERTIES OUTPUT_NAME "${_exe}_amrex")
```

Comment:
Wouldn't this change the executable name from `foo.exe` to `foo_amrex`? i.e. drop the `.exe` suffix?

Reply:
The old name does not have `.exe` either.

Reply:
I think using prefix here is better because all the amrex files will then be grouped together by default (by `ls` for example).

Reply:
The output name is before the executable suffix in cmake. It will keep .exe on Windows and none on Unix.